cockatrice/src/tab_message.cpp

```diff
@@ -116,6 +116,8 @@ void TabMessage::processUserMessageEvent(const Event_UserMessage &event)
         soundEngine->playSound("private_message");
     if (settingsCache->getShowMessagePopup() && shouldShowSystemPopup(event))
         showSystemPopup(event);
+    if (QString::fromStdString(event.sender_name()).simplified() == "Servatrice")
```

Comment:
The only other thing I might do is to make "servatrice" name a
constant shared across the app. Maybe even configurable so woods could
set it to "WoogerWorks Server" and poixen could do his own, etc.

I can imagine users being "Who is this servatrice user messaging me?"

On 9/18/15, Zach <notifications@github.com> wrote:
>> @@ -116,6 +116,8 @@ void TabMessage::processUserMessageEvent(const
>> Event_UserMessage &event)
>>          soundEngine->playSound("private_message");
>>      if (settingsCache->getShowMessagePopup() &&
>> shouldShowSystemPopup(event))
>>          showSystemPopup(event);
>> +    if (QString::fromStdString(event.sender_name()).simplified() ==
>> "Servatrice")
>
> missed case, thanks for pointing it out
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/Cockatrice/Cockatrice/pull/1527/files#r39855102